debian/usr.bin.snap-confine

```diff
@@ -1,7 +1,7 @@
 # Author: Jamie Strandboge <jamie@canonical.com>
```

Comment:
debian/usr.bin.snap-confine doesn't represent the correct path now. This doesn't matter to apparmor but can lead to user confusion. Please rename to usr.lib.snapd.snap-confine. This will require changes to dh_apparmor in debian/rules and debian/snap-confine.install